src/toil_scripts/lib/files.py

```diff
@@ -0,0 +1,77 @@
+from contextlib import closing
+import os
+import errno
+import tarfile
+import shutil
+
+
+# FIXME: replace with bd2k.util.files.mkdir_p
+def mkdir_p(path):
+    """
+    It is Easier to Ask for Forgiveness than Permission
+
+    :param str path: path to directory to create
+    """
+    try:
+        os.makedirs(path)
+    except OSError as exc:
+        if exc.errno == errno.EEXIST and os.path.isdir(path):
+            pass
+        else:
+            raise
+
+
+def tarball_files(tar_name, fpaths, work_dir='.', prefix=''):
+    """
+    Creates a tarball from a group of files
+
+    :param str tar_name: Name of tarball
+    :param list[str] fpaths: File paths to include in the tarball
+    :param str work_dir: Current working directory
```

Comment:
It's the location of the tarball, not the current working directory.

The interaction between the actual current working directory, work_dir and the input paths (absolute or relative) worries me slightly. At the very least it should be documented.